src/c++/perf_analyzer/docs/llm.md

```diff
@@ -60,23 +60,14 @@ size on first-token latency. We issue single request to the server of fixed
 input sizes and request the model to compute at most one new token. This
 essentially means one pass through the model.
 
-#### 1. Run the following commands to set `max_tokens` to `1`
-
-```bash
-# in the `tutorials/Quick_Deploy/vLLM` directory from above
-PATH_TO_MODEL_PY="model_repository/vllm/1/model.py"
-MAX_TOKENS=1
-sed -i "128s/.*/\ \ \ \ \ \ \ \ params_dict[\"max_tokens\"] = ${MAX_TOKENS}/" ${PATH_TO_MODEL_PY}
-```
-
-#### 2. Start Triton Server
+#### (Optional) Start Triton Server if not already running
```

Comment:
Can this be under Setup section (including the one in the benchmark 2)? Since we only need to boot up server once.

Reply:
Moved 👍